sdk/js/__tests__/Client.spec.js

```diff
@@ -316,16 +326,28 @@ describe('DevCycleClient tests', () => {
             expect(window.localStorage.getItem(StoreKey.IdentifiedConfig)).toBe(
                 JSON.stringify(testConfig),
             )
+            expect(client.isInitialized).toBe(true)
         })
 
         it('should save config for anonymous user', async () => {
-            const client = new DevCycleClient('test_sdk_key', {
+            const client = new DevCycleClient(test_key, {
                 isAnonymous: true,
             })
             await client.onClientInitialized()
             expect(window.localStorage.getItem(StoreKey.AnonymousConfig)).toBe(
                 JSON.stringify(testConfig),
             )
+            expect(client.isInitialized).toBe(true)
+        })
+
+        it('should not initialize if there is an error reaching the DevCycle Client SDK API', async () => {
+            const client = new DevCycleClient(error_test_key, {
+                user_id: 'user1',
+            })
+            const callback = jest.fn()
+            const onInitialized = client.onClientInitialized(callback)
+            expect(onInitialized).toBeFalsy()
+            expect(client.isInitialized).toBe(false)
```

Comment:
i thought we wanted this to be true?

Reply:
ah the test description is wrong.. but this test is with a bad SDK Key, thus the `isInitialized` should be false

Reply:
thats not what its testing though, since youre just throwing a regular error